src/main/java/com/google/devtools/build/lib/authandtls/GoogleAuthUtils.java

```diff
@@ -217,12 +237,51 @@ public static CallCredentialsProvider newCallCredentialsProvider(@Nullable Crede
    */
   @Nullable
   public static Credentials newCredentials(@Nullable AuthAndTLSOptions options) throws IOException {
+    Optional<Credentials> credentials = newGoogleCredentials(options);
+
+    return credentials.orElse(null);
```

Comment:
Can we make `newGoogleCredentials` public and call it directly, avoiding this indirection? (It's fine if it returns a `@Nullable` for now; I'd rather fix all callers in a single CL than introduce temporary forwarding methods.)

Reply:
done